### ISSUE TITLE
Add support for IPv6

### ISSUE THREAD

**Issue by emil992** [rzander/sccmclictr#157]
Hi, is it possible to add support for connecting to clients over IPv6? Direct Access use IPv6 and is mostly used these days...

**Comment by hrrysckr** [comment]
Agreed! With our staff all working remotely with DA, it's frustrating not being able to use the tool as we had been able to whilst in the office.

**Comment by rzander** [comment]
Are you able to connect your devices with native PowerShell over IPv6 ? `Enter-PSSession {Hostname}`
